api/nakadi-event-bus-api.yaml

```diff
@@ -1186,6 +1186,29 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  SubscriptionCursor:
+    allOf:
+      - $ref: '#/definitions/Cursor'
+      - type: object
+        properties:
+          id:
+            type: string
+            format: uuid
+            description: |
+              Id of the cursor (a UUID).
+          event_type_name:
+            type: string
+            description: |
+              The name of the event type this partition's events belong to
+          cursor_token:
+            type: string
+            description: |
+              An opaque value defined by the server.
+        required:
+          - id
+          - event_type_name
+          - cursor_token
+
```

Comment:
hmmmm ..... what about the partition? without knowing the partition, `commit_at_will` is quite unpredictable, isn't it?